components/homme/src/theta-l_kokkos/prim_driver_mod.F90

```diff
@@ -3,6 +3,8 @@
 #endif
 module prim_driver_mod
 
+  use prim_driver_base,     only: deriv1, smooth_topo_datasets
```

Comment:
Do the mods of this sort fully resolve the text diffs you noted in #1816?

Reply:
Reposting the comment from above 
```Homme src files after this PR is in will mostly match between scream and e3sm. namelist mod and src/theta-l_kokkos/CMakeLists.txt do not match yet. outside of src, there are also differences.
```
So, the issue is only partially resolved.